content/en/_index.html

```diff
@@ -44,10 +44,6 @@ <h2>The Challenges of Migrating 150+ Microservices to Kubernetes</h2>
         <br>
         <br>
         <br>
-        <a href="https://www.lfasiallc.com/events/kubecon-cloudnativecon-china-2018/" button id="desktopKCButton">Attend KubeCon in Shanghai on Nov. 13-15, 2018</a>
-        <br>
-        <br>
-        <br>
         <br>
         <a href="https://events.linuxfoundation.org/events/kubecon-cloudnativecon-north-america-2018/" button id="desktopKCButton">Attend KubeCon in Seattle on Dec. 11-13, 2018</a>
```

Comment:
Since you're here, could you please change the date `Dec. 11-13` to `Dec. 10-13` as the related event page shows?

Reply:
Guess we want to merge this PR before Dec. 13, otherwise it will be outdated again :)

Reply:
Just removed a few more Shanghai mentions and changed the Seattle dates.